tests/monitoring_test.go

```diff
@@ -287,6 +287,11 @@ var _ = Describe("Prometheus Alerts", func() {
 			vmName := createResources(true, false)
 			waitForSeriesToBeDetected(fmt.Sprintf("kubevirt_ssp_vm_rbd_volume{name='%s', rxbounce_enabled='false'}", vmName))
 			waitForAlertToActivate("VirtualMachineCRCErrors")
+
+			Eventually(func() error {
```

Comment:
Get VM and check for error is not found instead?

Reply:
updated